book/website/community-handbook/acknowledgement/acknowledgement-members.md

```diff
@@ -0,0 +1,33 @@
+# Community Members and Shared Ownership
+
+## Community members, contributors and co-authors
+
+As a community-led project, we believe in and promote a collaborative approach to develop the content of this book.
+Writing a chapter is an iterative process that involves designing, outlining, writing, reviewing, and editing its contents.
+Similarly, community-oriented processes evolve over time with the help and feedback from the community.
+Everyone who contributes to this book, no matter how small or big their contributions are, is recognised in this project as a contributor and a community member.
+
+All the substantial contributors are named as co-authors on the book as a whole.
```

Comment:
What's the process for determining what a substantial contribution is, who is involved in that? Can that process itself be changed?

Reply:
In the example subchapter, we explain what kind of contributions are counted towards authorship. However, I agree that this word doesn't help and we should explain what we mean by "substantial".

Reply:
I just saw that you actually make it more precise in the `GOVERNANCE.md`:
> "Substantial" is a subjective term. As the project is quite small at the moment, the lead investigator Dr Kirstie Whitaker, will decide whether a contribution or collection of contributions is substantial enough to be given authorship. She can be contacted by email at kwhitaker@turing.ac.uk.

Perhaps it make sense to link that here! Of course there is still the question on how and under what conditions that decision process could be changed and whether some other decision process may be more appropriate. I don't have definite answers to these questions and can't find much guidance on them either, but they seem to be important, no?

Reply:
I mean not just the decision whether something is "substantial" or not but more generally how decision are made.

Reply:
Good point. I am still working on this chapter and can add a section on community process and workflow - something that is currently missing.

Reply:
Reading more carefully now. I think I will draft the process and workflow chapter separately. There are a few more community aspects that I need to document where we will have more on governance. Though, that should definitely be cross referenced here.

Reply:
We're in the process of drafting a governance document in sktime, perhaps that's helpful, feedback is of course also appreciated :) https://github.com/alan-turing-institute/sktime/pull/324